### PR TITLE
Adding delft_fiat

### PR DESCRIPTION
**Pull request by dalmijn**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by dalmijn** [comment]
I'm happy to maintain this package.

**Comment by dalmijn** [comment]
@conda-forge/help-python, could you review my recipe?

File: recipes/delft_fiat/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python >=3.9
+    - pip
+  run:
+    - gdal >=3.5
+    - numpy
+    - python >=3.9
```
Comment (ocefpaf) [inline recipes/delft_fiat/meta.yaml:26]: Metadata has py37

```suggestion
requirements:
  host:
    - python >=3.7
    - pip
  run:
    - gdal >=3.5
    - numpy
    - python >=3.7
```

File: recipes/delft_fiat/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+test:
+  imports:
+    - fiat
+  requires:
+    - pip
+  commands:
+    - pip check
+
+about:
+  home: https://github.com/simplejson/simplejson
```
Comment (ocefpaf) [inline recipes/delft_fiat/meta.yaml:39]: Copy-n-pasta. Please use [grayskull](https://github.com/conda/grayskull) to generate the recipes to help with that.
```suggestion
  home: https://github.com/Deltares/Delft-FIAT
```

File: recipes/delft_fiat/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5be537ec4d21cd3d452050e11aa40ed35fc2343bd9d61609a21858adaea42749
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - fiat = fiat.cli.main:main
+  script: {{ PYTHON }} -m pip install . -vv
```
Comment (ocefpaf) [inline recipes/delft_fiat/meta.yaml:17]:
```suggestion
  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
```

File: recipes/delft_fiat/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,49 @@
+{% set name = "delft_fiat" %}
+{% set version = "0.1.0rc1" %}
```
[the remark this answers is not in the document]
Comment (dalmijn) [reply recipes/delft_fiat/meta.yaml:2]: There is no stable release yet. That is planned in the upcoming months.

File: recipes/delft_fiat/meta.yaml (hunk 1 above)
Comment (dalmijn) [reply recipes/delft_fiat/meta.yaml]: Done.

File: recipes/delft_fiat/meta.yaml (hunk 3 above)
Comment (dalmijn) [reply recipes/delft_fiat/meta.yaml]: Done.

**Comment by dalmijn** [comment]
@ocefpaf It's ready for review again!

File: recipes/delft_fiat/meta.yaml (hunk 4 above)
[the remark this answers is not in the document]
Comment (dalmijn) [reply recipes/delft_fiat/meta.yaml:2]: Well, now I did.

[the remark this answers is not in the document]
Comment (dalmijn) [reply recipes/delft_fiat/meta.yaml:2]: @ocefpaf  When we do a stable release, the conda_build_config.yaml can just be deleted in a PR?

[the remark this answers is not in the document]
Comment (ocefpaf) [reply recipes/delft_fiat/meta.yaml:2]: Yep, delete and re-render.

[the remark this answers is not in the document]
Comment (dalmijn) [reply recipes/delft_fiat/meta.yaml:2]: Great!

**Pull request merged**